src/components/AccountList.vue

```diff
@@ -1,9 +1,12 @@
 <template>
-    <div class="flex flex-col bg-white rounded-lg shadow">
+    <div v-if="!busy" class="flex flex-col bg-white rounded-lg shadow">
         <div v-for="(user, index) in shownUsers" :key="user">
             <user-row :user="user" :is-first-row="index === 0" :is-last-row="index === shownUsers.length - 1" />
         </div>
     </div>
+    <div v-else>
+        <loading-spinner />
+    </div>
```

Comment:
```suggestion
    <div v-if="busy">
        <loading-spinner />
    </div>
    <div v-else class="flex flex-col bg-white rounded-lg shadow">
        <div v-for="(user, index) in shownUsers" :key="user">
            <user-row :user="user" :is-first-row="index === 0" :is-last-row="index === shownUsers.length - 1" />
        </div>
    </div>
```
positive case is more readable